Test/ProxyTestCase.php

```diff
@@ -72,15 +74,22 @@ protected function getHttpClient()
     /**
      * Get a response from your application through a live caching proxy
      *
-     * @param string $url     Request URL (absolute or relative)
+     * @param string $uri     Request URL (absolute or relative)
      * @param array  $headers Request HTTP headers
      * @param array  $options Request options
      *
      * @return Response
      */
-    protected function getResponse($url, array $headers = array(), $options = array())
+    protected function getResponse($uri, array $headers = array())
```

Comment:
it looks like this is never used. the only classes extending this are ProxyTestCaseTest which only wants the client and mocks everything, and ProxyTestCaseAnnotationTest which is empty. as in, we missed to implement anything in that test.